.github/workflows/release.yml

```diff
@@ -13,14 +13,26 @@ jobs:
     steps:
       - name: Checkout code
         uses: actions/checkout@v2
+        
+      - name: Read version.properties
+        id: version
+        uses: juliangruber/read-file-action@v1
+        with:
+          path: ./version.properties
+        
+      - uses: rishabhgupta/split-by@v1
+        id: split
+        with:
+          string: "${{ steps.version.outputs.content }}"
+          split-by: '='
```

Comment:
you might wanna try this action to read properties file instead https://dev.to/madhead/reading-java-properties-in-github-actions-2c2k

Reply:
Thanks @luongvo , let me try with this.

Reply:
@luongvo It works as well, just takes more time to run but less code.
Both versions work as expected. So which version we should choose?

Reply:
@doannimble reading those values as property name definitely better than argument index 😄 I would prefer to read it as property reading way instead of raw text
![image](https://user-images.githubusercontent.com/16315358/135595065-8a68034b-afe1-42b9-9230-4a60b4a07cb1.png)

Reply:
@luongvo I know that. Maybe that's the reason why it takes more time to complete the job.
I'm Ok to use the property.